### PR TITLE
Remove space between `(` followed by `{`

### PR DESCRIPTION
**Pull request by kimdv**

### CODE REVIEW

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by kimdv** [comment]
@swift-ci please test

File: CodeGeneration/Sources/generate-swiftsyntax/templates/basicformat/BasicFormatFile.swift [hunk 1]
```diff
@@ -246,15 +257,15 @@ let basicFormatFile = SourceFileSyntax {
...
-             (.postfixQuestionMark, .leftParen), // Ensures there is not space in `init?()` or `myOptionalClosure?()`s
-             (.postfixQuestionMark, .rightAngle), // Ensures there is not space in `ContiguousArray<RawSyntax?>`
-             (.postfixQuestionMark, .rightParen): // Ensures there is not space in `myOptionalClosure?()`
+        case (.exclamationMark, .leftParen), // Ensures there is not a space in `myOptionalClosure!()`
+             (.exclamationMark, .period), // Ensures there is not a space in `myOptionalBar!.foo()`
+             (.keyword(.as), .exclamationMark), // Ensures there is not a space in `as!`
+             (.keyword(.as), .postfixQuestionMark), // Ensures there is not a space in `as?`
+             (.keyword(.try), .exclamationMark), // Ensures there is not a space in `try!`
+             (.keyword(.try), .postfixQuestionMark), // Ensures there is not a space in `try?`:
+             (.postfixQuestionMark, .leftParen), // Ensures there is not sa pace in `init?()` or `myOptionalClosure?()`s
```
Comment (ahoppen) [inline CodeGeneration/Sources/generate-swiftsyntax/templates/basicformat/BasicFormatFile.swift:266]: typo

```suggestion
             (.postfixQuestionMark, .leftParen), // Ensures there is not a space in `init?()` or `myOptionalClosure?()`s
```

**Review by ahoppen — APPROVED** [review]
Sounds reasonable to me

**Comment by kimdv** [comment]
@swift-ci plese test

**Comment by kimdv** [comment]
@swift-ci please test

**Pull request merged**